src/components/Slider/Slider.jsx

```diff
@@ -0,0 +1,242 @@
+      indicateSelection,
+      prefix,
+      postfix
+    },
+    ref
+  ) => {
+    const componentRef = useRef(null);
+    const mergedRef = useMergeRefs({ refs: [ref, componentRef] });
+    const consumerBem = createBemBlockHelper(classNameBase, { isConsume: true });
+    const providerProps = {
+      ariaLabel,
+      ariaLabeledBy,
+      classNameBase,
+      color,
+      dataTestId,
+      disabled,
+      isRange,
+      max,
+      min,
+      onChange,
+      showValue,
+      size,
+      step,
+      value,
+      valueDefault: ensureValueDefault({ valueDefault, isRange, min, max }),
+      valueFormatter,
+      valueText,
+      infixOptions: {
+        prefix,
+        postfix,
+        indicateSelection
+      }
+    };
+    return (
+      <SliderProvider {...providerProps}>
+        <div
+          className={bem("", { disabled, "value-shown": showValue }, className)}
+          data-testid={dataTestId}
+          id={id}
+          ref={mergedRef}
+        >
+          <SliderInfix kind="prefix" />
+          <SliderBase className={consumerBem("base")} />
+          <SliderInfix kind="postfix" />
+        </div>
+      </SliderProvider>
+    );
+  }
+);
+
+Slider.sizes = SIZES_BASIC;
+Slider.colors = COLORS_BASIC;
+
+Slider.propTypes = {
+  // ------ SliderBase props
+  /**
+   * Define a string that labels the current element (Slider)
+   */
+  ariaLabel: PropTypes.string,
+  /**
+   * ElementId of Node that have the text needed for labeling the current element (Slider)
+   */
+  ariaLabeledBy: PropTypes.string,
+  /**
+   * Custom `class name` to be added to the component-root-node
+   */
+  className: PropTypes.string,
+  /**
+   * Custom base of class names for add to component-root and internal-components nodes
+   *  - check Dev Tools for found available Custom/Consumer Classes
+   */
+  classNameBase: PropTypes.string,
+  /**
+   * Color Mode (primary/positive/negative) of the component (Slider)
+   */
+  color: PropTypes.oneOf(Object.values(Slider.colors)),
+  /**
+   * Unique TestId - can be used as Selector for integration tests and other needs (tracking, etc)
+   */
+  "data-testid": PropTypes.string,
+  /**
+   * If set to true, Component (Slider) will be disabled
+   *  - impossible to change value of component (Slider)
+   *  - visual changes (opacity)
+   */
+  disabled: PropTypes.bool,
+  /**
+   * Attribute `id` to be added to the component-root-node
+   */
+  id: PropTypes.string,
+  /**
+   * If true switch slider to RRange mode (two Thumbs)
+   */
+  isRange: PropTypes.bool,
+  /**
+   * Max range value of the component (Slider)
+   */
+  max: PropTypes.number,
+  /**
+   * Min range value of the component (Slider)
+   */
+  min: PropTypes.number,
+  /**
+   * Optional onChange callback (for outer trigger or Controlled mode)
+   * - required in Controlled Mode
+   */
+  onChange: PropTypes.func,
+  /**
+   * Step for Component (Slider) changes
+   */
+  step: PropTypes.number,
```

Comment:
I did not understand what this prop does. I tried to play with it in the controls and nothing happens. what it should do?

Reply:
let's keep this prop but add more clear comment about what is purpose is?

Reply:
I'm changing description.
step it pretty common property for sliders:
https://mui.com/api/slider/
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_slider_role

Reply:
Done!